frontend/src/pages/Publish.js

```diff
@@ -38,9 +41,18 @@ export default function Publish() {
     event.preventDefault()
     createFlat(flat, token)
       .then(setFlat)
-      .finally(() => setFlat(initialState))
+      .finally(() => setFlat(initialFlat))
```

Comment:
Finally will always run, so you set the flat after successfully creating it and then immediately set the form to the initial state? I think one of the two is enough, depending on your userflow

Reply:
You're right, one is enough... Tnx